indico/modules/events/editing/client/js/selectors.js

```diff
@@ -199,3 +199,10 @@ export const getLastRevertableRevisionId = createSelector(
     return lastFinalRev.id;
   }
 );
+
+export const canJudgeLastRevision = createSelector(
+  getLastRevision,
+  lastRevision => {
```

Comment:
I think this would fit in one line using the `=> foo` style instead of `=> { return foo; }`